tests/test_other.py

```diff
@@ -7851,9 +7839,9 @@ def test(filename, expectations, size_slack):
 
       print('test on hello world')
       test(path_from_root('tests', 'hello_world.cpp'), [
-        ([],      21, ['assert'], ['waka'], 46505,  22,   15, 58), # noqa
```

Comment:
why did this increase? I thought this just moved code around, but left the same imports and exports from the asm module?

Reply:
Oh, sloppy __deps handling, had that included also in ALLOW_MEMORY_GROWTH=0 builds, fixed now.